mesa/experimental/jupyter_viz.py

```diff
@@ -13,6 +13,9 @@
 plt.switch_backend("agg")
 
 
+model_parameters = solara.reactive(None)
```

Comment:
Having this as a global variable is not portable for `JupyterViz`.

Reply:
removed